content/en/data_streams/_index.md

```diff
@@ -13,170 +17,21 @@ Data Streams Monitoring provides a standardized method for teams to understand a
 * Pinpoint faulty producers, consumers or queues, then pivot to related logs or clusters to troubleshoot faster.
 * Prevent cascading delays by equipping service owners to stop backed up events from overwhelming downstream services.
 
-## Setup
+### Setup
 
-{{< programming-lang-wrapper langs="java,go,dotnet" >}}
+| Runtime | Supported technologies |
+|---|----|
+| Java | Kafka (self-hosted, Amazon MSK), RabbitMQ, HTTP, gRPC |
+| Go | All (through [manual instrumentation][1]) |
+| .NET | Kafka (self-hosted, Amazon MSK) |
+| Other | See [Agent installation requirements][2] |
```

Comment:
Maybe we remove 'Other' for now, and we can do this in a separate PR? Essentially, we want 'Other' to link to public docs that cover something like what is in this Google Doc: https://docs.google.com/document/d/1tlZvdEZPEgV-8-645ZV6Ijhs-t-C7m4F0IBAYBrluQo/edit